provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -1704,6 +1759,9 @@ func populateRosaClassicClusterState(ctx context.Context, object *cmv1.Cluster,
 	state.State = types.StringValue(string(object.State()))
 	state.Name = types.StringValue(object.Name())
 	state.CloudRegion = types.StringValue(object.Region().ID())
+	if state.AdminCredentials.IsUnknown() {
+		state.AdminCredentials = adminCredentialsNull()
+	}
```

Comment:
This is not needed here because in the createCluster Null is assigned as a default value

Reply:
@nirarg 
if w/o this, in update scenario, I will get the error `Attribute admin_credentials, cannot be changed from <null> to <unknown>`.